moonstone/analysis/statistical_test.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import pandas as pd
+from scipy import stats
+from statsmodels.stats.multitest import multipletests
+from typing import List, Union
+
+from moonstone.utils.pandas.series import SeriesBinning
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def statistical_test_groups_comparison(series: pd.Series, group_series: pd.Series, stat_test: str,
+                                       output: str = 'dataframe', mirror: str = True, **kwargs):
+    """
+    :param output: {'series', 'dataframe'}
+
+    In kwargs, you can pass argument for statistical test, like :
+    :param equal_var: For ttest_ind, set to True if your samples have the same variance and
+    you wish to perform a Student's t-test rather than a Welch's t-test (here default is False)
+    :param alternative: {None, 'two-sided', 'less', 'greater'} For Mann - Whitney-U, you can define
+    the alternative hypothesis.
+    :param bins: For chi2_contingency, you can define bins.
+    """
+
+
+    possibles = globals().copy()
```

Comment:
I do not really get that, I think for this, you should use a simpler approach either by calling directly the function corresponding to the method selected or use a class with the `getattr`.

Reply:
It was a solution to remove the eval that I found on stackoverflow. I should you conditions? I was trying to avoid having a test for every iterations but it shouldn't be too time consuming anyway

Reply:
I think conditions would be easier to read. I will add another global comment on the function.